.github/workflows/phase_3_yocto.yml

```diff
@@ -0,0 +1,194 @@
+---
+name: Phase 3 - Yocto
+on:
+  push:
+    paths:
+      - .github/workflows/phase_3_yocto.yml
+
+env:
+  YOCTO_TAG: styhead
+  PARLAY_VERSION: 0.6.0
+  SBOMASM_VERSION: 0.1.5
+  SBOMQS_VERSION: 0.1.9
+  TRIVY_VERSION: 0.54.1
+
+jobs:
+  Generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Environment
+        run: |
+          sudo apt update
+          sudo apt install -y gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 python3-subunit zstd liblz4-tool file locales libacl1
+          pip3 install websockets
+          sudo locale-gen en_US.UTF-8
+      - name: Checkout Yocto
+        run: |
+          git clone git://git.yoctoproject.org/poky
+          cd poky
+          git checkout ${YOCTO_TAG}
+      - name: Build Yocto
+        run: |
+          cd poky
+          source oe-init-build-env
+          echo "BB_NUMBER_THREADS=\"8\"" >> conf/local.conf
+          echo "PARALLEL_MAKE=\"-j 8\"" >> conf/local.conf
+          echo "INHERIT += \"rm_work\"" >> conf/local.conf
+          echo "BB_HASHSERVE = \"auto\"" >> conf/site.conf
```

Comment:
If you want more human readable SPDX JSON output, you can add:

```shell
echo "SPDX_PRETTY = "\1\"" >> conf/local.conf
```

But otherwise it looks good to me